server/entity/Media.ts

```diff
@@ -164,6 +167,8 @@ class Media {
         this.ratingKey
       }`;
 
+      this.iOSPlexUrl = `plex://preplay/?metadataKey=%2Flibrary%2Fmetadata%2F${this.ratingKey}&server=${machineId}`;
```

Comment:
This shouldn't be set unless there is a rating key, similar to `plexUrl`.

Reply:
They are in the same if statement.